docs/variants/pc_engines/firmware-update.md

```diff
@@ -0,0 +1,24 @@
+# Legacy -> UEFI Update
```

Comment:
UEFI -> Legacy would also be appreciated.

Reply:
I has not been exercised here, since UEFI firmware is not released yet.